source/class/qx/theme/manager/Font.js

```diff
@@ -243,6 +247,15 @@ qx.Class.define("qx.theme.manager.Font", {
         return qx.bom.webfonts.WebFont;
       }
       return qx.bom.Font;
+    },
+
+    /**
+     * Returns the font information output by the compiler
+     * @internal subject to change
+     * @return {Object}
+     */
+    getManifestFonts() {
```

Comment:
If this is internal, should it be public? Maybe protected? Or prefixed with '$'?